glide-core/src/client/mod.rs

```diff
@@ -459,14 +459,22 @@ impl Client {
     /// Unlike a transaction, the commands are not executed atomically, and in cluster mode, the commands can be sent to different nodes.
     ///
     /// The `raise_on_error` parameter determines whether the pipeline should raise an error if any of the commands in the pipeline fail, or return the error as part of the response.
-    /// The `retry_failed_commands` parameter determines whether the pipeline should retry failed commands that are retryable, this can result on re-ordering of commands in the same slots.
-    /// Note that redirect errors will still be redirected, regardless of the value of `retry_failed_commands`.
+    /// - `pipeline_retry_strategy`: Configures the retry behavior for pipeline commands.
+    ///   - If `retry_server_error` is `true`, failed commands with a retriable `RetryMethod` will be retried,
+    ///     potentially causing reordering within the same slot.
+    ///     ⚠️ **Caution**: This may lead to commands being executed in a different order than originally sent,
+    ///     which could affect operations that rely on strict execution sequence.
+    ///   - If `retry_connection_error` is `true`, sub-pipeline requests will be retried on connection errors.
+    ///     ⚠️ **Caution**: Retrying after a connection error can lead to duplicate executions, as it is unclear
```

Comment:
Caution: Retrying after a connection error may result in duplicate executions, since the server might have already received and processed the request before the error occurred.